### PR TITLE
marco to support switch profile

### PR DESCRIPTION
**Pull request by Voomlz**
Swap profile using a macro:
/myslot load profileName

### CODE REVIEW

**Comment by tg123** [comment]
this is really cool, allow me some validation in weekend

**Comment by Voomlz** [comment]
I don't know if you could get the options in the saved data and use those instead

File: gui.lua [hunk 1]
```diff
@@ -484,7 +484,40 @@ end)
...
+            MyslotExports = {}
+        end
+        if not MyslotExports["exports"] then
+            MyslotExports["exports"] = {}
+        end
+        local exports = MyslotExports["exports"]
+
+        for i, profile in ipairs(exports) do
+            --MySlot:Print(L["profile " .. profile.name])
+            if profile.name == what then
```
Comment (tg123) [inline gui.lua:500]: Lets print a message when not found
then LGTM

thanks

**Comment by Voomlz** [comment]
Updated.

**Pull request merged**